src/ralph/cli.py

```diff
@@ -38,12 +38,11 @@
 # cli module logger
 logger = logging.getLogger(__name__)
 
-# Lazy evaluations
-DATABASE_BACKENDS = (lambda: [backend.value for backend in DatabaseBackends])()
-PARSERS = (lambda: [parser.value for parser in Parsers])()
-STORAGE_BACKENDS = (lambda: [backend.value for backend in StorageBackends])()
-STREAM_BACKENDS = (lambda: [backend.value for backend in StreamBackends])()
-BACKENDS = (lambda: DATABASE_BACKENDS + STORAGE_BACKENDS + STREAM_BACKENDS)()
+DATABASE_BACKENDS = [backend.value for backend in DatabaseBackends]
+PARSERS = [parser.value for parser in Parsers]
+STORAGE_BACKENDS = [backend.value for backend in StorageBackends]
+STREAM_BACKENDS = [backend.value for backend in StreamBackends]
+BACKENDS = DATABASE_BACKENDS + STORAGE_BACKENDS + STREAM_BACKENDS
```

Comment:
Why removing lazy evaluation?

Reply:
I'm not very sure, but I would like to question whether these are lazy evaluations.
Aren't the `()` at the end eagerly evaluating the lambda functions?

Reply:
Outch. Yes. I have a better suggestion, WDYT?

```suggestion
DATABASE_BACKENDS = map(lambda b: b.value, DatabaseBackends)
PARSERS = map(lambda p: p.value, Parsers)
STORAGE_BACKENDS = map(lambda b: b.value, StorageBackends)
STREAM_BACKENDS = map(lambda b: b.value, StreamBackends)
BACKENDS = DATABASE_BACKENDS + STORAGE_BACKENDS + STREAM_BACKENDS
```

Reply:
I like the Idea to use `map` instead) Thanks)
Just wrapped it inside a lambda to be able to iterate over it multiple times.
However, pylint now complains about assigning lambda functions to variables.
Updated this part [in the last commit](https://github.com/openfun/ralph/pull/184/commits/4896806bd4aaddf541178eca94e74e73e169ad5b).